packages/ocean-react/src/Chart/types/DoughnutChart.types.ts

```diff
@@ -0,0 +1,41 @@
+import React, { ReactNode } from 'react';
```

Comment:
The ocean-web components pattern has the `components types`  in the same file as the component.